README.md

```diff
@@ -109,13 +109,36 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
 
 ### Optional configuration options
 
-| Environment variable                  | Default           | Description
-| --------------------------------------| ----------------  | -----------
-| ENABLE_KAFKA                          | false             | Feature flag to send kafka messages when a collection is published
-| KAFKA_ADDR                            | localhost:9092    | Comma seperated list of kafka brokers
-| KAFKA_CONTENT_PUBLISHED_TOPIC         | content-published | The name of the topic to send event to
-| KAFKA_SEC_PROTO                       | _unset_           | if set to "TLS", kafka connections will use TLS
-| KAFKA_SEC_CLIENT_KEY_P12              | _unset_           | if using TLS (see above), this is a base64-encoded PKCS12 keystore (optional, used for client auth)
+| Environment variable                              | Default                                     | Description
+| --------------------------------------------------| --------------------------------------------| -----------
+| DEFAULT_WEBSITE_URL                               | "http://localhost:8080"                     | Service URL
+| DEFAULT_SLACK_WARNING_CHANNEL                     | "slack-client-test"                         | Slack channel
+| DEFAULT_SLACK_ALARM_CHANNEL                       | "slack-client-test"                         | Slack alarm channel
+| DEFAULT_SLACK_USERNAME                            | "Zebedee"                                   | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                        | "http://localhost:8080"                     | Service public URL
+| DEFAULT_FLORENCE_URL                              | "http://localhost:8081"                     | Florence URL
+| DEFAULT_BRIAN_URL                                 | "http://localhost:8083"                     | Brian URL
+| DEFAULT_TRAIN_URL                                 | "http://localhost:8084"                     | Train URL
+| DEFAULT_DYLAN_URL                                 | "http://localhost:8085"                     | Dylan URL
+| CONTENT_DIRECTORY                                 | "content"                                   | Content directory
+| KAFKA_SEC_PROTO                                   | _unset_                                     | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_KEY_P12                          | _unset_                                     | if using TLS (see above), this is a base64-encoded PKCS12 keystore (optional, used for client auth)
```

Comment:
I didn't see this var being used in the code, where is it used?
Instead, I saw the following vars being used if kafka tls is enabled:
- KAFKA_SEC_CLIENT_KEY
- KAFKA_SEC_CLIENT_CERT

I think `KAFKA_SEC_CLIENT_KEY_P12` should be removed and  `KAFKA_SEC_CLIENT_KEY` and `KAFKA_SEC_CLIENT_CERT` should be added, unless I am missing something